CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to the Imperative package will be documented in this file.
 
+## `4.9.1`
```

Comment:
This should be `## Recent Changes` (https://github.com/zowe/imperative/blob/master/CONTRIBUTING.md#changelog-update-guidelines) - sorry for the troubles

Reply:
Thanks. I just tried to follow the existing structure with numbers.

Reply:
yeah - it's not clear... maybe we can put a hidden note in the changelog itself in the future